sourcegraph/cody
# [Cody Web]: Support `openctx` providers

Closes https://linear.app/sourcegraph/issue/SRCH-596/remote-repositories
Closes https://linear.app/sourcegraph/issue/SRCH-576/new-ui-added-openctx-which-has-dependencies-on-the-dom-so-it-doesnt

This PR adds support for open context providers, so we would be able to mention remote repositories and remote files in Cody Web. It tries to not touch logic outside of the scope, changes should have no effect on VSCode or general cody agent logic.

## Test plan
- Run cody web demo 
- Check that you can use remote repositories and remote files providers mentions (which is called just repositories and files when it's running for Cody Web)
